recipes/pytorch-pretrained-bert/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pytorch-pretrained-bert" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-','_') }}-{{ version }}.tar.gz
+  sha256: a1bbcf9d2ef5f894b583737bd0bcbd7f4cbffddc4722b255aaf8b93fd43892d3
+  patches:
+    - 0001-use-entry-points
+
+build:
+  skip: True  # [not linux or py<35]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - pytorch-pretrained-bert=pytorch_pretrained_bert.__main__:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pytorch >=0.4.1
+    - numpy
+    - boto3
+    - requests
+    - tqdm
+
+test:
+  imports:
+    - pytorch_pretrained_bert
+
+about:
+  home: https://pypi.org/project/pytorch-pretrained-bert/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
See https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
I think in CB3, conda will look for the license in both places. Is there a particular reason why we chose to use the environment one?

Reply:
I don't think that is true... but I could be wrong.

Reply:
I found the commit where that feature was added:

https://github.com/conda/conda-build/commit/85c46be3bdbd4dc13cc9be0b6db45c1eeac91d11

Is it worth it to update the documentation indicating that either mode is valid?